dotcom-rendering/src/web/components/BlockquoteBlockComponent.tsx

```diff
@@ -34,52 +34,92 @@ const quotedBlockquoteStyles = (palette: Palette) => css`
 	color: ${palette.text.blockquote};
 `;
 
-export const BlockquoteBlockComponent = ({ html, palette, quoted }: Props) => {
-	const {
-		willUnwrap: isUnwrapped,
-		unwrappedHtml,
-		unwrappedElement,
-	} = unwrapHtml({
-		fixes: [
-			{ prefix: '<p>', suffix: '</p>', unwrappedElement: 'p' },
-			{
-				prefix: '<blockquote>',
-				suffix: '</blockquote>',
-				unwrappedElement: 'blockquote',
-			},
-			{
-				prefix: '<blockquote class="quoted">',
-				suffix: '</blockquote>',
-				unwrappedElement: 'div',
-			},
-		],
-		html,
-	});
+const parseHtml = (html: string): DocumentFragment => JSDOM.fragment(html);
+
+// The nodeType for ELEMENT_NODE has the value 1.
+function isElement(node: Node): node is Element {
+	return node.nodeType === 1;
+}
```

Comment:
How is this different from `node instanceof Element`?

Reply:
Not entirely sure, I took this straight from AR cc @JamieB-gu

Reply:
Originally added in this PR: https://github.com/guardian/apps-rendering/pull/116

I think they probably provide the same functionality in this case @mxdvl ?

Reply:
One feels a lot more readable than the other in my opinion, but I'll let you guess which 😜

Reply:
> I think avoiding custom type guard might be a good reason to favour `instanceof`...

Yes, actually that would be preferable, I agree 👍. That means we could do away with this function and allow TypeScript to narrow it automatically for you without relying on the type predicate.

Reply:
Actually, thinking some more about this: that would work on the client, but how would that work server-side? `instanceof` operates on values, so it needs a value called `Element` available in the current scope.

Reply:
```ts
import { JSDOM } from 'jsdom';
const Element = new JSDOM().window.Element;
if (node instanceof Element) {
	console.log(node.nodeType === Element.prototype.ELEMENT_NODE)
}
```

Reply:
I think comparing that option to the original `isElement` type guard based on `nodeType`, I think the type guard may be preferable?